app/App.css

```diff
@@ -111,8 +112,14 @@ th {
 
 td {
   border-left: 2px solid #000000;
+  vertical-align: top !important;
```

Comment:
do you need !important?

It would be better to use a more specific selector, e.g. .results-table td. https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
fixed